src/Interface/Modules/Visualization/CreateStandardColorMap.ui

```diff
@@ -45,6 +45,16 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QLabel" name="label_7">
+        <property name="text">
+         <string>Hold shift to remove a point</string>
```

Comment:
Either have a period here or remove the period from the sentence above about alphas.